examples/cd_text_imdb.ipynb

```diff
@@ -6,20 +6,23 @@
    "source": [
```

Comment:
<p><code>The method works...</code>  sentence rendered as a title currently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>